client/proposalmsgs_test.go

```diff
@@ -70,7 +71,7 @@ func TestChannelProposalReqSerialization(t *testing.T) {
 			m, err = clienttest.NewRandomVirtualChannelProposal(rng, client.WithNonceFrom(rng), app)
 			require.NoError(t, err)
 		}
-		wire.TestMsg(t, m)
+		iotest.MsgSerializerTest(t, m)
```

Comment:
It's difficult to review if you change the same line several times within a single PR.

Reply:
I understand, I will try to minimize this next time.